subprojects/docs/src/docs/userguide/authoring-builds/incremental_builds.adoc

```diff
@@ -499,7 +501,7 @@ The `TaskInputs.files()` method returns a builder that has a `skipWhenEmpty()` m
 Now that you have seen both the annotations and the runtime API, you may be wondering which API you should be using. Our recommendation is to use the annotations wherever possible, and it’s sometimes worth creating a custom task class just so that you can make use of them. The runtime API is more for situations in which you can’t use the annotations.
 
 [[sec:runtime_api_for_custom_tasks]]
-==== Using it for custom task types
+==== Registering inputs and outputs for custom task types
```

Comment:
❌ I think we should end up with one verb for "declaring" inputs and outputs for tasks. Currently, this page uses "manage", "register" and "declare". Or even something else as well?

Reply:
Let's use "declare"